resources/js/components/Board.js

```diff
@@ -88,16 +91,24 @@ export class Board extends React.Component {
     );
   }
 
+  renderFolderPath = () => {    
+    return this.state.folderPath.map(folder =>
+      <p className="folder-path"> {folder} {'>'}</p>
```

Comment:
if you just want to display inline text, you can use a `<span>` tag instead without any extra CSS instead of a `<p>`
```suggestion
      <span>{folder} {'>'}</span>
```